bsp/stm32/stm32l4r9-st-eval/board/ports/drv_touch.c

```diff
@@ -0,0 +1,208 @@
+/*
+ * File      : drv_touch.c
+ * This file is part of RT-Thread RTOS
```

Comment:
Please take care of license issue. These files should be license under Apache License v2.0